pallets/asset/src/lib.rs

```diff
@@ -1857,14 +1856,16 @@ impl<T: Trait> Module<T> {
         funding_round: Option<FundingRoundName>,
     ) -> DispatchResult {
         ensure!(
-            name.len() <= T::AssetNameMaxLength::get(),
+            name.len() as u32 <= T::AssetNameMaxLength::get(),
```

Comment:
Super nit: I'd do the reverse, cast on the other side to `usize` cause its more likely to narrow the limit and be more strict rather than truncate the length and be more lenient. Although the numbers here are so big that it doesn't matter in practice, so you can probably just ignore this comment. xD

Reply:
Leaving this as it is for now. I started refactoring it a bit but noticed we are using `as u32` on len() and comparing it to u32 quite a lot in the codebase and sometimes in parity's code. It became not worth the effort after that discovery xD